lightning/src/offers/refund.rs

```diff
@@ -102,8 +102,8 @@ pub struct RefundBuilder {
 }
 
 impl RefundBuilder {
-	/// Creates a new builder for a refund using the [`Refund::payer_id`] for signing invoices. Use
-	/// a different pubkey per refund to avoid correlating refunds.
+	/// Creates a new builder for a refund using the [`Refund::payer_id`] for the node to send to
```

Comment:
Hmm... pushed a fixup but, IIRC, I chose `payer_id` because it seemed more general (and what is used in the spec) whereas `node_id` is specific to the "no paths" case.

Did you mean to change it only on `RefundBuilder::new` or throughout the file?